spec/latest/core/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Hydra Core Vocabulary</title>
 <meta charset="utf-8">
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch|gated"></script>
```

Comment:
How is this better if it requires an additional dependency (at least in some browsers) and makes more HTTP requests (in all browsers)?